AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/backgroundfilter/BackgroundFilterProcessor.swift

```diff
@@ -89,7 +89,7 @@ public class BackgroundFilterProcessor {
             logger.info(msg: "Updating buffer pool with new sizes: \(inputFrameCG.width) x \(inputFrameCG.height)")
             updateBufferPool(newWidth: inputFrameCG.width, newHeight: inputFrameCG.height)
             // Initialize the segmentationProcessor if it has not been initialized.
-            let initializeResult: Bool = segmentationProcessor.initialize(height: segmentationProcessorHeight,
```

Comment:
Does this still considering breaking since the criteria is whether builder needs to update their code to adapter the upgrade?

Reply:
I think so. I think it should be called breaking in the changelog and next release might need to bump up the version.